po/pl.po

```diff
@@ -480,45 +540,89 @@ msgstr "Przeniesiono zrzut ekranu do kosza"
 msgid "Screenshots moved to trash"
 msgstr "Przeniesiono zrzuty ekranu do kosza"
 
-#: gradia/ui/image_creation/source_image_generator.py:539
+#: gradia/ui/image_creation/source_image_generator.py:554
 msgid "Source Snippet From %Y-%m-%d %H-%M-%S"
-msgstr "Fragmenty źródłowe od %Y-%m-%d %H-%M-%S"
+msgstr "Fragmenty źródłowe z %Y-%m-%d %H-%M-%S"
+
+#: gradia/ui/widget/quick_color_picker.py:84
+msgid "Red"
+msgstr "Czerwony"
+
+#: gradia/ui/widget/quick_color_picker.py:85
+msgid "Green"
+msgstr "Zielony"
+
+#: gradia/ui/widget/quick_color_picker.py:86
+msgid "Blue"
+msgstr "Niebieski"
+
+#: gradia/ui/widget/quick_color_picker.py:87
+msgid "Yellow"
+msgstr "Żółty"
+
+#: gradia/ui/widget/quick_color_picker.py:88
+msgid "Purple"
+msgstr "Fioletowy"
+
+#: gradia/ui/widget/quick_color_picker.py:93
+msgid "Black"
+msgstr "Czarny"
+
+#: gradia/ui/widget/quick_color_picker.py:94
+msgid "White"
+msgstr "Biały"
+
+#: gradia/ui/widget/quick_color_picker.py:114
+msgid "More colors..."
+msgstr "Więcej kolorów..."
+
+#: gradia/ui/widget/quick_color_picker.py:196
+msgid "Choose Color"
+msgstr "Wybierz kolor"
+
+#: gradia/ui/widget/aspect_ratio_button.py:26 data/ui/image_sidebar.blp:81
+msgid "Aspect Ratio"
+msgstr "Proporcje"
+
+#: gradia/ui/widget/aspect_ratio_button.py:38
+msgid "Free"
+msgstr "Darmowe"
```

Comment:
Not sure if this is the correct word. "Free" in this context is used to indicate that no aspect ratio is applied to the cropping:

<img width="127" height="274" alt="Image" src="https://github.com/user-attachments/assets/2db305f0-ef0d-4a99-81ed-623bde379ff0" />